lib/guardian.ex

```diff
@@ -767,6 +782,25 @@ defmodule Guardian do
     apply(mod, :config, [:token_module, @default_token_module])
   end
 
+  @doc false
```

Comment:
It might make sense to move this to https://github.com/ueberauth/guardian/blob/master/lib/guardian/token/jwt.ex and do a small refactor to avoid a bit of code dublication

Reply:
Well.. it's funny you should say this, as a version of this logic did previously exist in jwt.ex, I necessarily decoupled the `Map.put` stuff in `assign_exp_from_ttl` from the actual ttl parsing to avoid the duplication, and then moved it out of jwt.ex to guardian.ex as I viewed it as a general utility function; it didn't seem to have anything really to do with the JWT token implementation at all as such at that point... and calling code from jwt.ex to parse a config option just didn't seem quite right to me... I'll move it to jwt.ex if you like though.

Reply:
I think you are right, that it should actually live in `guardian`, and we should just refactor `jwt` to use the code from `guardian` and not the other way around